server/hedley/modules/custom/hedley_reports/hedley_reports.module

```diff
@@ -3687,7 +3687,7 @@ function hedley_reports_well_child_generate_vaccination_progress_data_regular(ar
     }
 
     foreach ($immunisations_by_type[$type]['doses'] as $index => $dose) {
-      $common_type = hedley_reports_drop_preffix_and_suffix($type, 'well_child_', '_immunisation');
+      $common_type = hedley_reports_drop_prefix_and_sufix($type, 'well_child_', '_immunisation');
```

Comment:
This is tricky.
https://ies.ed.gov/ncee/rel/regions/southeast/pdf/prefix_base_word_suffix.pdf
`prefix`
and `suffix` are the correct forms AFAIK.